scripts/bot/run-bot.js

```diff
@@ -22,12 +23,19 @@ const querystring = require("query-string");
   page.on("error", err => console.error("ERROR: ", err));
   page.on("pageerror", err => console.error("PAGE ERROR: ", err));
 
+  const baseUrl = options["--url"] || `https://${options["--host"]}/hub.html`;
```

Comment:
do you want to throw an error/warning here if `options["--host"]` is missing?

Reply:
Host defaults to `"localhost:8080"`.